CONTRIBUTING.md

```diff
@@ -227,15 +227,17 @@ To explore DevTools with all experimental features enabled:
 
 1. [Configure](https://docs.flutter.dev/get-started/install) Dart or Flutter.
 
-2. Start DevTools:
+2. Create an empty folder and `cd` to it in terminal. 
+
+3. Start DevTools:
 ```
-git clone git@github.com:flutter/devtools.git
+git clone https://github.com/flutter/devtools.git
```

Comment:
It might be worth leaving both options since both are vlid
```suggestion
- Clone with SSH
    - git clone git@github.com:flutter/devtools.git
- OR clone with https
    - git clone https://github.com/flutter/devtools.git
```

Reply:
I want the script to be simple and not requiring choices and edits, so that UX people could get devtools up and running fast.
Professional engineers will know how to deviate if they want.
I am choosing http, because SSH requires configuration.